RSEAUNZAccessibility.Rmd

```diff
@@ -0,0 +1,334 @@
+accessibility communication. Potential participants will need 
+these answers before deciding whether to submit proposals
+for presentations or to attend the conference. In the future, 
+this information should be available when open submissions are 
+first announced. Questions about accessibility should be 
+answered promptly and not leave the participant wondering
+if their query reached anyone who can find answers or fix
+problems.  Don't charge late fees if answers could not be 
+provided before early registration closes.
+
+## Zoom
+
+Most important features of Zoom are accessible to people with disabilities. 
+Zoom has prioritized accessibility more than many other platforms. 
+It is in wide use, so some people will be quite familiar with it. 
+Others may be less experienced and take more time to find their way around the interface.
+
+Conference presentations rely on the assumption
+that the audience receives information on two channels: they look at 
+the slide to appreciate some of the information, while listening
+to the author explain the content. 
+The synthesis of the information from
+these two channels leads to comprehension of the presentation. 
+For example, in a presentation about statistical analysis code
+the slides are likely to contain the precise code while 
+the verbalized part of the talk might be more general or 
+explanatory. For people with sensory disabilities, there 
+are accessibility practices to provide the missing information.  
+
+Accurate captions are critical for the participation of people who are deaf or 
+hard-of-hearing. They also benefit people who are logging in
+from noisy environments or whose primary language is not English. Zoom 
+provides automatic captions at no cost, however, they are not accurate
+enough for specialized or technical language. They cannot properly recognize 
+vocabulary like software package and function names, code, and 
+terms related to data science or statistics. (Deaf people call these
+"craptions.") 
+
+### Suggestions
+
+Captions should be provided by humans who have an 
+opportunity to familiarize themselves with the field in advance. There are 
+many services that provide these "live captions." One that has been effective in the R
+community is [White Coat Captioning](https://whitecoatcaptioning.com).
+
+Since some deaf people use sign language as their first language, it can
+be an ideal accessibility practice. Interpreters should also have an opportunity to become familiar with the field and its vocabulary. One 
+complication with sign language interpretation is that each country
+has its own sign language. Since this conference is focused on Australia, 
+Auslan interpretation should be considered.
+
+Both live captions and sign language interpreters can be integrated easily 
+into Zoom presentations. The Zoom environment should be set up 
+so the user can make the presenter, the slides, and the interpreter visible
+(not small images).
```

Comment:
Question: Is this still possible with a full-screen presentation (other than enabling gallery mode and then positioning the speaker and the interpreter suitably)?